nautobot/docs/docker/index.md

```diff
@@ -63,15 +48,15 @@ Nautobot requires a PostgreSQL database and Redis instance before it will start.
 Most configuration parameters are available via environment variables which can be passed to the container.  If you desire you can inject your own `nautobot_config.py` by overriding `/opt/nautobot/nautobot_config.py` using [docker volumes](https://docs.docker.com/storage/volumes/) by adding `-v /local/path/to/custom/nautobot_config.py:/opt/nautobot/nautobot_config.py` to your docker run command, for example:
 
 ```no-highlight
-$ docker run --name nautobot -v /local/path/to/custom/nautobot_config.py:/opt/nautobot/nautobot_config.py ghcr.io/nautobot/nautobot
+$ docker run --name nautobot -v /local/path/to/custom/nautobot_config.py:/opt/nautobot/nautobot_config.py nautobot/nautobot
```

Comment:
Same here and for the rest of the references to `nautobot/nautobot` as the image.